core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -144,6 +148,15 @@
   }
 
   /* All classes below should start with .conversation-skin */
+  .conversation-skin-back-to-collection-container {
+    margin-top: 25px;
```

Comment:
I think this is the same for mobile, but I can not find an easy way to test this. I only have one card, and I would rather not make another collection, but if it comes down to it... :(